browser-extension/src/models/banner/BannerOptions.js

```diff
@@ -0,0 +1,24 @@
+export class BannerOptions {
+  constructor(
+    TS_BANNER_DATE_START,
+    TS_BANNER_DATE_END,
+    TS_BANNER_ACTION_FR_URL,
+    TS_BANNER_ACTION_EN_URL,
+    TS_BANNER_PROOF_KW
+  ) {
+    this.TS_BANNER_DATE_START = new Date(TS_BANNER_DATE_START);
+    this.TS_BANNER_DATE_END = new Date(TS_BANNER_DATE_END);
+    this.TS_BANNER_ACTION_FR_URL = TS_BANNER_ACTION_FR_URL;
+    this.TS_BANNER_ACTION_EN_URL = TS_BANNER_ACTION_EN_URL;
+    this.TS_BANNER_PROOF_KW = TS_BANNER_PROOF_KW;
+  }
+}
+
+// TODO: these values are placeholder values that should be updated.
+export const defaultBannerOptions = new BannerOptions(
+  '2020-01-01T00:00:00Z',
+  '2022-01-01T00:00:00Z',
```

Comment:
The study is now in progress, let's not forget to update those values.

See: https://github.com/tournesol-app/tournesol/commit/daa91a0175777158012a559e9777c245dfc60d8e

Reply:
solved by [7426da6](https://github.com/tournesol-app/tournesol/pull/1436/commits/7426da6081f4634f2c78b87540d3daa3216ad861)